lib/ansible/plugins/httpapi/checkpoint.py

```diff
@@ -63,7 +63,7 @@ def send_request(self, path, body_params):
 
             return response.getcode(), self._response_to_json(value)
         except AnsibleConnectionFailure as e:
-            return 404, 'Object not found'
```

Comment:
have u tested this exception case, and is it working as expected?

Reply:
@justjais I tested it, but it's not working. I don't get the complete error of checkpoint..

Reply:
@chkp-orso if the error is not complete, it's better to have generic statements like `return 404, 'Object not found'`, rather than having incomplete error.